sdks/hermes-engine/hermes-engine.podspec

```diff
@@ -75,6 +77,9 @@ Pod::Spec.new do |spec|
       #{File.exist?(import_hermesc_file) ? "export HERMES_OVERRIDE_HERMESC_PATH=#{import_hermesc_file}" : ""}
       #{File.exist?(import_hermesc_file) ? "echo \"Overriding HermesC path...\"" : ""}
 
+      # Remove this when Hermes properly support multi core builds
+      "$NODE_BINARY" #{fix_script} $(pwd)/utils/build-apple-framework.sh
+
       # Build iOS framework
       ./utils/build-ios-framework.sh
```

Comment:
Commented on the WG Discussion: I would rather make sure we run the script on React Native end's rather than patching the one coming from Hermes.